### ISSUE TITLE
could this project be used in cyber range?

### ISSUE THREAD

**Issue by RunnerHong** [OpenIxia/bps_restpy#3]
I want to use BreakingPoint VE to simulate attack traffic in cyber range, cloud I use this api to could control the BreakingPoint VE? something like trigger traffic. thanks.

**Comment by nutu** [comment]
Yes this is exactly what is it for. You can do 99 percent from the actions done in the UI.
